src/main/java/uk/gov/digital/ho/hocs/workflow/BpmnService.java

```diff
@@ -274,6 +274,12 @@ public void updateAllocationNote(String caseUUIDString, String stageUUIDString,
         log.info("Adding Casenote to Case: {}", caseUUIDString);
     }
 
+    public void updateCaseNote(String caseUUIDString, String caseNote, String caseNoteType) {
+        log.debug("######## Save Case Note ########");
```

Comment:
Would be useful for these comments to be updated to state it's updating rather than saving, will help debugging problems in the future.

Reply:
I will change this to show what is really happening. I used this terminology to sympathise with what is already used in CaseworkClient!! which is not always accurate.